test/controllers/admin/comfy/users_controller_test.rb

```diff
@@ -26,6 +26,31 @@ class Comfy::Admin::UsersControllerTest < ActionDispatch::IntegrationTest
     assert response.body.include? I18n.t('views.comfy.users.index.header.action')
   end
 
+  test "#index: users table with all necessary columns is rendered" do
+    column_headings = [
+      "Email",
+      "Name",
+      "Can manage web",
+      "Can manage analytics",
+      "Can manage files",
+      "Can manage email",
+      "Can manage users",
+      "Can manage blog",
+      "Can manage api",
+      "Can manage subdomain settings",
+      "Can view restricted pages",
+      "Can manage forum",
+      "Current sign in at",
+      "Last sign in at"
+    ]
+    sign_in(@user)
+    get admin_users_url(subdomain: @domain)
+    assert_select "table", 1, "This page must contain a users table"
+    column_headings.each do |heading|
+      assert_select "thead th", {count: 1, text: heading}, "Users table must contain '#{heading}' column"
```

Comment:
@Ayon95 nice tests! let the Ruby flow through you 🔴

Reply:
Thanks!